proto/api_spec/method.proto

```diff
@@ -318,6 +318,15 @@ message HTTPMethodMeta {
     v3 = 3;
   }
   PathParameterInferenceAlgorithm path_param_alg = 5;
+
+  enum Obfuscation {
+    // No obfuscation is applied.
+    NONE = 0;
+
+    // The value is replaced with zero value.
+    ZERO_VALUE = 1;
```

Comment:
These values must be flipped. Existing witnesses will default to 0, which according to this, will be interpreted as no obfuscation.